packages/main/types/requests.ts

```diff
@@ -99,7 +107,17 @@ export interface StartChatParams extends BaseParams {
  * @public
  */
 export interface CountTokensRequest {
-  contents: Content[];
+  generateContentRequest?: GenerateContentRequest;
+  contents?: Content[];
+}
+
+/**
+ * Params for calling {@link GenerativeModel.countTokens}
```

Comment:
This comment should probably be updated.

Reply:
Done.